core/src/net/sf/openrocket/file/openrocket/importt/DocumentConfig.java

```diff
@@ -359,6 +359,8 @@ class DocumentConfig {
 		setters.put("InnerTube:clusterconfiguration", new ClusterConfigurationSetter());
 		setters.put("InnerTube:clusterscale", new DoubleSetter(
```

Comment:
Can you add a comment above this line: `// TODO: delete clusterscale when compatibility with OpenRocket 15.03 is not needed anymore`